CHANGES.md

```diff
@@ -1,10 +1,15 @@
 # Changes
 
 ## Unreleased - 2020-xx-xx
-* Implement Logger middleware regex exclude pattern [#1723]
 
+* Implement Logger middleware regex exclude pattern [#1723]
```

Comment:
undo the diff noise

Reply:
-> 06841e111c58a30537455fb2e0ba8e72df0ffcfe